cmd/kube-apiserver/app/server.go

```diff
@@ -438,33 +456,24 @@ func parseRuntimeConfig(s *options.APIServer) (map[string]genericapiserver.APIGr
 	}
 	_ = disableLegacyAPIs // hush the compiler while we don't have legacy APIs to disable.
 
-	// "api/v1={true|false} allows users to enable/disable v1 API.
```

Comment:
The changes between here until the `for key := range s.RuntimeConfig {` loop-- what is the purpose?